docs/src/data/mockServices.tsx

```diff
@@ -0,0 +1,78 @@
+let signUpPassword = '';
+const verifiedUsers = new Map();
+const FIXED_USERNAME = 'test';
+const FIXED_PASSWORD = 'password';
+
+export const mockServices = {
+  async getAmplifyConfig() {
+    return {};
+  },
+  async handleSignUp(formData) {
+    signUpPassword = formData.password;
+    return {};
+  },
+  async checkVerifiedContact() {
+    return { verified: {}, unverified: {} };
+  },
+  async handleSignIn({
+    username,
+    password,
+  }: {
+    username: string;
+    password: string;
+  }) {
+    if (
+      verifiedUsers.get(username).password === password ||
+      (username === FIXED_USERNAME && password === FIXED_PASSWORD)
+    ) {
+      return Promise.resolve(verifiedUsers.get(username));
+    } else {
+      return Promise.reject(new Error('Invalid username or password'));
+    }
+  },
+  async handleConfirmSignUp({
+    username,
+    code,
+  }: {
+    username: string;
+    code: string;
+  }) {
+    if (code === '123456') {
+      verifiedUsers.set(username, {
+        username: username,
+        password: signUpPassword,
+      });
```

Comment:
To make this easier to read, we should move the second argument in its own variable, example:
```
const input =  { username, password: signUpPassword ];

verifiedUsers.set(username, input);
```

Reply:
Done!